client/wfprev-war/src/main/angular/src/app/app.routing.ts

```diff
@@ -26,10 +26,23 @@ const PANEL_ROUTES: Routes = [
     component: ErrorPageComponent,
     pathMatch: 'full',
   },
-
-  { path: '', 
-    redirectTo: ResourcesRoutes.MAP, 
-    pathMatch: 'full' } // Default route to map
+  {
+    path: ResourcesRoutes.EDIT_PROJECT,
+    loadChildren: () =>
```

Comment:
Shouldn't the authentication be getting checked on the edit project page as well?

Reply:
added